src/Sandboxing.php

```diff
@@ -250,31 +250,49 @@ public function finalize_document( Document $dom, $effective_sandboxing_level )
 
 		$this->remove_required_amp_markup_if_not_used( $dom, $effective_sandboxing_level );
 
-		$amp_admin_bar_menu_item = $dom->xpath->query( '//div[ @id = "wpadminbar" ]//li[ @id = "wp-admin-bar-amp" ]/a' )->item( 0 );
+		$amp_admin_bar_menu_item = $dom->xpath->query( '//div[ @id = "wpadminbar" ]//li[ @id = "wp-admin-bar-amp" ]' )->item( 0 );
 		if ( $amp_admin_bar_menu_item instanceof Element ) {
-			$span = $dom->createElement( Tag::SPAN );
-			$span->setAttribute(
-				Attribute::TITLE,
-				sprintf(
-					/* translators: %d is the effective sandboxing level */
-					__( 'Effective sandboxing level: %d', 'amp' ),
-					$effective_sandboxing_level
-				)
-			);
+
 			switch ( $effective_sandboxing_level ) {
 				case 1:
-					$text = '1️⃣';
+					$text  = '1️⃣';
+					$title = __( 'Sandboxing level: Loose (1)', 'amp' );
 					break;
 				case 2:
-					$text = '2️⃣';
+					$text  = '2️⃣';
+					$title = __( 'Sandboxing level: Moderate (2)', 'amp' );
 					break;
 				default:
-					$text = '3️⃣';
+					$text  = '3️⃣';
+					$title = __( 'Sandboxing level: Strict (3)', 'amp' );
 					break;
 			}
-			$span->textContent = $text;
-			$amp_admin_bar_menu_item->appendChild( $dom->createTextNode( ' ' ) );
-			$amp_admin_bar_menu_item->appendChild( $span );
+
+			$amp_link = $dom->xpath->query( './a', $amp_admin_bar_menu_item )->item( 0 );
+			if ( $amp_link instanceof Element ) {
+				$span = $dom->createElement( Tag::SPAN );
+				$amp_link->appendChild( $dom->createTextNode( ' ' ) );
+				$span->textContent = $text;
+				$span->setAttribute( Attribute::TITLE, $title );
+				$amp_link->appendChild( $span );
```

Comment:
Could this be reordered?
https://github.com/ampproject/amp-wp/blob/21412aae5eb1872ea801987bb5b8236eed1bc361/src/Sandboxing.php#L273-L277

To something like:
```
$span = $dom->createElement( Tag::SPAN );
$span->textContent = $text;
$span->setAttribute( Attribute::TITLE, $title );

$amp_link->appendChild( $dom->createTextNode( ' ' ) );
$amp_link->appendChild( $span );
```

Reply:
Yeah, except let's put the setAttribute call before textContent assignment. Then we don't have to align the equals signs 😊

Can you make your change an actual GitHub suggestion?

Reply:
> Can you make your change an actual GitHub suggestion?

Looks like you were faster 😁